Sources/Player/PlaybackEngine/Internal/Asset.swift

```diff
@@ -8,9 +8,9 @@ open class Asset {
 	let player: GenericMediaPlayer
 	private var loudnessNormalizationConfiguration: LoudnessNormalizationConfiguration
 
-	@Atomic var assetPosition: Double
+	@Atomic var assetPosition: Double = 0
 	@Atomic var playbackMetadata: AssetPlaybackMetadata?
-	@Atomic private var cached: Bool = false
+	@Atomic var cacheState: AssetCacheState?
```

Comment:
This is the only change not behind the FF but the change in the Legacy implementation to this new interface is very minor.